src/login/styles/_modals.scss

```diff
@@ -14,33 +14,52 @@
 }
 
 
+.modal-title {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  & .close-button-container{
+    display: flex;
+    justify-content: flex-end;
+  }
+  & span {
+    padding: 0 2.25rem 1.5rem 2.25rem;
+  }
+}
+
 
 h5.modal-title {
-  font-family: "Akkurat";
-  line-height: 1.2;
+  font-family: "Proxima Nova";
```

Comment:
I'm not sure that this Proxima Nova in the heading is working. The way our fonts are structured at the moment they have the weight in the name. I think it might be missing here.
<img width="500" alt="Screenshot 2020-10-28 at 10 54 37" src="https://user-images.githubusercontent.com/30963614/97421101-67666d80-190c-11eb-827a-e17704e45ec7.png">